docs/DeveloperGuide.md

```diff
@@ -775,6 +805,41 @@ testers are expected to do more *exploratory* testing.
 
 ***
 
+### Adding a student
+
+1. Adding a student
+
+    1. Prerequisites: Delete any student named `Alice Lee` before and between test cases. Ensure there is a student named `Alex Yeoh` in the list of students.
+
+    2. **Valid Test Cases**
+
+        1. Add a student with only compulsory fields:`add n/alice lee p/91234567 e/91234567 a/123 Clementi`<br>
+           Expected: New student with name `Alice Lee`, phone and emergency number `91234567`,
+           and address `123 Clementi` is added. There are no notes, tasks, subjects or lesson times stated.
```

Comment:
No level, subject(s) or lesson time(s) are specified.
emergency contact number (follow UG)

add does not handle tasks and notes